tests/lib/RichText/RendererTest.php

```diff
@@ -1720,67 +1720,57 @@ protected function getMockedRenderer(array $methods = [])
             ->getMock();
     }
 
-    /**
-     * @var \Ibexa\Contracts\Core\Repository\Repository|\PHPUnit\Framework\MockObject\MockObject
-     */
+    /** @var \Ibexa\Contracts\Core\Repository\Repository&\PHPUnit\Framework\MockObject\MockObject */
     protected $repositoryMock;
```

Comment:
I know it was untouched, but we can adjust it by the way I think:
```suggestion
    protected Repository $repositoryMock;
```